examples/crossplane-xr-multiversion/demo.sh

```
@@ -36,6 +55,69 @@ forward=()
+    --create-repo)
+      CREATE_REPO=1
+      if [[ "${2:-}" != "" && "${2:-}" != -* ]]; then
+        CREATE_REPO_NAME="$2"
+        shift 2
+      else
+        shift
+      fi
+      ;;
+    --create-repo=*)
+      CREATE_REPO=1
+      CREATE_REPO_NAME="${1#*=}"
```

Shellcheck (0.11.0)

[warning] CREATE_REPO_NAME appears unused. Verify use (or export if used externally).

(SC2034)